src/Pages/Register.js

```diff
@@ -1,117 +1,134 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState } from 'react';
```

Comment:
removed unnecessary imports and commented out functionality that will be applied later